app/controllers/books_controller.rb

```diff
@@ -0,0 +1,15 @@
+class BooksController < ApplicationController
+  include Wor::Paginate
+
+  def show
+    @book = Book.find(params[:id])
+    render json: @book, serializer: BookSerializer
+  rescue StandardError => e
+    render json: { error: "El libro con el id #{e.id} no existe" }, status: :not_found
```

Comment:
Don't do this. There might be another problema going on in the back that you are not noticing. If you are gonna use rescue, capture a specific exception.